roles/api-environment/templates/express-api/start-express-container.sh.j2

```diff
@@ -40,36 +25,29 @@ if [[ "$(docker ps -aq -f name=$api_name)" ]]; then
     done
 fi
 
-# Remove image if it already exists
-image_id=$(docker images -q $api_name)
-if [[ $image_id ]]; then
-  echo "Remove existing image $image_id"
-  docker rmi $image_id --force
-fi
-
 # Create logs folder and make sure the API have access to wrtie logs into it
 mkdir -p $api_path/logs
 chown -R {{ api_user }}:{{ api_group }} $api_path/logs
 
-# Load docker image from the artifact
-docker load --input $archive
-
 # Run API in a container
 uid="$(id -u {{ api_user }})"
 gid="$(id -g {{ api_user }})"
 docker run -d \
            --user "$uid:$gid" \
            --workdir /usr/src/$api_name \
+           --env API_NAME=$api_name \
+           --env FOREVER_ROOT=/usr/src/$api_name/.forever \
+           --env HOME=/usr/src/$api_name \
            --env-file $env \
-           --volume /apis/keytool_files:/usr/src/$api_name/keytool_files \
-           --volume $api_path/logs:/usr/src/$api_name/logs \
+           --volume /apis/keytool_files:/usr/src/$api_name/keytool_files:ro \
            --volume $config:/usr/src/$api_name/config:ro \
+           --volume $api_path:/usr/src/$api_name \
```

Comment:
Since `/usr/src/$api_name` is used in a few places, could you store it in a variable similar to `api_path` and `env` above?